solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -248,17 +248,6 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
         maxStakedBalanceEnabled = false;
     }
 
-    /**
-      * @dev returns the pool token address by the reserve token address
-      *
-      * @param _reserveToken    reserve token address
-      *
-      * @return pool token address
-    */
-    function poolToken(IERC20Token _reserveToken) public view returns (ISmartToken) {

```

Comment:
this is a public function that we still need to expose

Reply:
Fixed.